tsdb/head.go

```diff
@@ -1665,7 +1665,7 @@ func (s *memSeries) chunksMetas() []chunks.Meta {
 	return metas
 }
 
-// reset re-initialises all the variable in the memSeries except 'lset', 'ref',
+// reset re-initializes all the variable in the memSeries except 'lset', 'ref',
```

Comment:
This is not really a typographical error.

Reply:
well learned, thank you!